charts/frontend/templates/varnish-deployment.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.varnish.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-varnish
+  labels:
+    {{- include "frontend.release_labels" . | nindent 4 }}
+    service: varnish
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "frontend.release_selector_labels" . | nindent 6 }}
+      service: varnish
+  template:
+    metadata:
+      labels:
+        {{- include "frontend.release_labels" . | nindent 8 }}
+        service: varnish
+        silta-ingress: allow
+      annotations:
+        # We use a checksum to redeploy the pods when the configMap changes.
+        configMap-checksum: {{ include (print $.Template.BasePath "/varnish-configmap-vcl.yaml") . | sha256sum }}
```

Comment:
What about checksum of varnish-secret.yaml, should be added here?

Reply:
good idea!